pages/1.11/release-notes/1.11.0/index.md

```diff
@@ -0,0 +1,138 @@
+- DCOS_OSS-1489 - Add support for cgroup blkio statistics in `dcos-metrics`.
+- DCOS_OSS-2003 - Modify DC/OS overlay networking to work with systemd networkd.
+
+## <a name="fixed-issues"></a>Improvements and Major Issues Fixed in 1.11.0 Release Candidate 4
+- COPS-2201 - Fix `dcos-diagnostics` crashing after running gen_resolvconf.py.
+- COPS-2465 - License reporting is inconsistent. [enterprise type="inline" size="small" /]
+- DCOS-19500 - Fix cosmos/service/update failure when Marathon is deploying.
+- DCOS-20064 - Verify Azure and AWS advanced templates work with the licensing parameter. [enterprise type="inline" size="small" /]
+- DCOS-20343 - Fix UI loading screen.
+- DCOS-20396 - Test that bootstrap regenerates cert when old cert does not match new CSR.
+- DCOS-20485 - Adjust implementation to address performance issues.
+- DCOS-20492 - Cluster sometimes comes up without license when using AWS CloudFormation templates. [enterprise type="inline" size="small" /]
+- DCOS-20515 - Make licensing version-aware to prevent upgrade failures. [enterprise type="inline" size="small" /]
+- DCOS-20569 - Include only public attributes of license on diagnostics bundle. [enterprise type="inline" size="small" /]
+- DCOS-20628 - `dcos-vault`: fix deadlock in go-zookeeper library.
+- DCOS-20676 - `csidevices`: fix lsblk parsing of empty string bool values.
+- DCOS-20679 - Make licensing handle a cluster license renewal for same major version during a config upgrade. [enterprise type="inline" size="small" /]
+- DCOS-20772 - Update znode creation scheme for storing license audit data.
+- DCOS-21000 - Release Marathon plugin with resource full auth bug fix.
+- DCOS-21045 - Update `dcos-cluster-linker` to not allow a self-link. [enterprise type="inline" size="small" /]
+- DCOS-21095 - Turn IPv6 overlay off in IPv4 environment.
+- DCOS_OSS-1587 - Support DC/OS install on non-root LVM volume.
+- DCOS_OSS-2070 - Service is not accessible via l4lb VIP from local agent using UCR bridge mode.
+- DCOS_OSS-2105 - UI can show stale task state due to dropped events (Related MESOS-8469 - Fix the problem with Mesos master dropping events in the operator API stream.)
+- METRONOME-100 - Metronome restart causes duplication of jobruns.
+- METRONOME-190 - Add authorized launch queue.
+- METRONOME-191 - Implement start deadline timeout.
+- METRONOME-194 - Support FORBID concurrency policy.
+- Updated marathon to 1.6.322 (https://github.com/dcos/dcos/pull/2473 | https://github.com/mesosphere/dcos-enterprise/pull/2278)
+- Updated Mesos to 1.5.x 9840ae1 (https://github.com/dcos/dcos/pull/2472 | https://github.com/mesosphere/dcos-enterprise/pull/2263)
+
+## <a name="fixed-issues"></a>Improvements and Major Issues Fixed in 1.11.0 Release Candidate 1
+- DCOS-16368 -  DC/OS Enterprise Secret Store no longer accepts secrets with uppercase letters in their paths. Secret paths with uppercase letters conflicted with use of the IAM API. [enterprise type="inline" size="small" /]
```

Comment:
quick note: this is probably incorrect

Reply:
What should this say instead?

Reply:
I think we should remove it altogether. The change in question was reverted. See [1].

[1] https://github.com/mesosphere/dcos-docs-site/pull/649#discussion_r172832231